.github/workflows/ci.yaml

```diff
@@ -20,26 +20,17 @@ jobs:
     name: Audit Rust vulnerabilities
     runs-on: ubuntu-latest
     steps:
-    - name: Install Rust toolchain
-      uses: actions-rs/toolchain@v1
-      with:
-        toolchain: stable
-        profile: minimal
-        override: true
-
     - name: Checkout repository
       uses: actions/checkout@v2
 
-    - run: sudo chown -R $(whoami):$(id -ng) ~/.cargo/
-
-    - name: Cache cargo
-      uses: actions/cache@v1
+    - uses: actions-rs/install@v0.1
       with:
-        path: ~/.cargo/
-        key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+        crate: cargo-audit
+        use-tool-cache: true
 
-    - run: cargo install cargo-audit
-    - run: cargo audit
+    - uses: actions-rs/audit-check@v1
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why do we need a token here?

Reply:
Looks like it's used for [reporting](https://github.com/actions-rs/audit-check/blob/master/src/main.ts#L67-L82).

Reply:
Ah, the action wants to send a PR... I'd rather not allow it, let the build fail, and send the required code change myself:

* I am not really comfortable about giving tokens to 3rd party actions, given that the action code can be updated at any time. 
* Security updates seem like things where careful manual review has a lot of benefits, and is only rarely required

Reply:
See also https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token#using-the-github_token-in-a-workflow.

Reply:
Not a PR, I think it either files an issue or displays some UI in the checks pane. But I guess we don't really need that..

Reply:
Updated.

Reply:
but won't the action just die without the token? The code doesn't check for `== null`, and readme implies that it is non-optional.

Reply:
I removed the action. Are you suggesting that we use it, but not pass the token and let it crash? I'm not sure it brings much value, since it won't be able to display [the nice UI](https://user-images.githubusercontent.com/3797062/64926127-b8b0bc00-d834-11e9-97d5-5b6aa06dc573.png).

Reply:
Oups, sorry, forgot to actually read the new diff :sweat_smile: 

bors r+

Reply:
> bors r+

I don't think that does anything :smile_cat:.

Reply:
If I trie hard enough it will work some day,right?